src/assets/css/twitterwall.css

```diff
@@ -1,147 +1,149 @@
-.background{
-	list-style:none;
-	display:inline-block;
-	margin: 0;
-	padding:0;
-	border:1px solid black;
+.background {
+  list-style: none;
+  display: inline-block;
+  margin: 0;
+  padding: 0;
+  border: 1px solid black;
 
 }
-.main-container{
-	min-width: 1114px;
-	height:auto;
-	background-color: #333;
-	margin-right:3%;
-	padding: 1%;
-	min-height:432px;
+.main-container {
+  min-width: 1114px;
+  height: auto;
+  background-color: #333;
+  margin-right: 3%;
+  padding: 1%;
+  min-height: 432px;
+
 }
-.item-wrapper{
-	opacity: .7;
+.item-wrapper {
+  opacity: .7;
+
 }
-.tweet{
-	float:left;
-	width:50%;
+.tweet {
+  float: left;
+  width: 50%;
 
-	
 }
-.image{
-	box-sizing: border-box;
-	width: 50%;
-    padding: 4%;
-    float:left;
+.image {
+  box-sizing: border-box;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The box-sizing property isn't supported in IE6 and IE7.](https://www.codacy.com/app/triplez/open-event-webapp/file/2434084751/issues/source?bid=3233857&fileBranchId=3238420#l28)